src/main/kotlin/com/carneirofatima/vehicleterminal/controller/ControllerAdvice.kt

```diff
@@ -0,0 +1,24 @@
+package com.carneirofatima.vehicleterminal.controller
+
+import com.carneirofatima.vehicleterminal.InvalidBodyException
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+@ControllerAdvice
```

Comment:
This is nice! Add a generic Exception :
```
@ExceptionHandler(Exception::class)
fun handleException(exception: Exception) { 
   //
}
```
This allow to lead with unexpected exceptions